src/androidTest/java/com/nutomic/syncthingandroid/test/syncthing/RestApiTest.java

```diff
@@ -75,7 +73,7 @@ public void onReceiveSystemInfo(RestApi.SystemInfo info) {
     public void testGetFolders() {
         assertNotNull(mApi.getFolders());
     }
-    
+
     public void testConvertNotCrashing() {
```

Comment:
Please revert your changes to this file. Of course the whitespace and extra imports shouldn't be here, but removing that should go into a seperate cleanup commit/PR.